### PR TITLE
Arnej/outward conn check

### PR DESCRIPTION
**Pull request by arnej27959**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

@havardpe please review

### CODE REVIEW

**Comment by arnej27959** [comment]
@havardpe please take another look

File: configd/src/apps/sentinel/outward-check.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    } else if (req->GetErrorCode() == FRTE_RPC_NO_SUCH_METHOD ||
+               req->GetErrorCode() == FRTE_RPC_WRONG_PARAMS)
+    {
+        LOG(debug, "Connected OK to %s but no reverse connectivity check available", _spec.c_str());
+        _result = CcResult::REVERSE_UNAVAIL;
+    } else {
+        LOG(debug, "error on request to %s : %s (%d)", _spec.c_str(),
+            req->GetErrorMessage(), req->GetErrorCode());
+        _result = CcResult::CONN_FAIL;
+
```
Comment (havardpe) [inline configd/src/apps/sentinel/outward-check.cpp:46]: empty line

File: configd/src/apps/sentinel/outward-check.cpp [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        if (answer == "ok") {
+            LOG(debug, "ping to %s with reverse connectivity OK", _spec.c_str());
+            _result = CcResult::ALL_OK;
+        } else {
+            LOG(debug, "connected to %s, but reverse connectivity fails: %s",
+                _spec.c_str(), answer.c_str());
+            _result = CcResult::REVERSE_FAIL;
+        }
+    } else if (req->GetErrorCode() == FRTE_RPC_NO_SUCH_METHOD ||
+               req->GetErrorCode() == FRTE_RPC_WRONG_PARAMS)
```
Comment (havardpe) [inline configd/src/apps/sentinel/outward-check.cpp:38]: consider including FRTE_RPC_WRONG_RETURN here as well (matches wrong params)

**Review by havardpe — APPROVED** [review]
looks good

**Pull request merged**